assets/scss/utilities/_hide.scss

```diff
@@ -28,9 +28,18 @@
 }
 
 .u-hidden\@medium {
-    display: block;
+    display: flex;
 
     @include bp(medium) {
         display: none;
     }
 }
+
+
+.u-hidden\@mobile {
+    display: none;
+
+    @include bp(medium) {
+        display: flex;
```

Comment:
`u-hidden@mobile` shouldn't overwrite styles when the element is visible. It will potentially break styling for elements that have different `display` property e.g. `inline` or `grid` 

Use `bp-down` instead